core/src/main/java/org/mapstruct/ValueMappings.java

```diff
@@ -11,6 +11,9 @@
 import java.lang.annotation.Target;
 
 /**
+ * // TODO I didn't find the direct explanation in documentation,
```

Comment:
Is the same as for `@Mappings`.. (only pre-java-8). So I guess that needs to be mentioned.